rust/agama-dbus-server/src/questions.rs

```diff
@@ -16,6 +16,16 @@ impl GenericQuestionObject {
         self.0.id
     }
 
+    #[dbus_interface(property)]
+    pub fn class(&self) -> &str {
```

Comment:
Please update also `doc/dbus/org.opensuse.Agama.Questions1.Generic.doc.xml`

Reply:
will do, my plan is to update doc and test when everything works together as I am not sure if we not need to modify more.